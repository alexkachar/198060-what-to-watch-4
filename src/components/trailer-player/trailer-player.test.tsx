import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TrailerPlayer from './trailer-player';

import mockMovies from '../../test-data/mock-movies';

const {previewImage, previewVideoLink} = mockMovies[0];

it(`TrailerPlayer renders correctly`, () => {
  const tree = renderer
    .create(
        <TrailerPlayer
          isPlaying={true}
          previewImage={previewImage}
          previewVideoLink={previewVideoLink}
        />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
