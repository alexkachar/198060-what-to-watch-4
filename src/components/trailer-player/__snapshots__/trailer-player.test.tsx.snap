// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrailerPlayer renders correctly 1`] = `
<video
  height="175"
  muted={true}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  poster="img/the-grand-budapest-hotel.jpg"
  src="https://some-link"
  width="280"
/>
`;
